packages/lesswrong/components/linkPreview/PostLinkPreview.tsx

```diff
@@ -203,7 +202,7 @@ const PostLinkPreviewWithPost = ({classes, href, innerHTML, post, id, error}: {
 
   if (!post) {
     return <span {...eventHandlers}>
-      <Link to={href}  dangerouslySetInnerHTML={{__html: innerHTML}}/>;
+      <Link to={href}  dangerouslySetInnerHTML={{__html: innerHTML}}/>
```

Comment:
This is the key line, this added a semicolon to in-process link renders.

![](https://user-images.githubusercontent.com/9090789/81350150-30f31880-9076-11ea-9f3d-55f13dc17ed0.png)

![](https://user-images.githubusercontent.com/9090789/81350140-29cc0a80-9076-11ea-9492-9ec801b456f6.png)